TLM/TLM/Custom/PathFinding/CustomPathFind_Old.cs

```diff
@@ -0,0 +1,3725 @@
+﻿#define DEBUGLOCKSx
```

Comment:
file can be deleted

Reply:
It is used when you build for Benchmark. Removing it also means we can remove PF2 define.
Should I proceed and remove?

Reply:
File deleted, `PF2` renamed to `PF_DIJKSTRA`. I will try and add A⭐️ in the future, so there will be `PF_ASTAR` or something.